plutus-tx/src/PlutusTx/Builtins/HasOpaque.hs

```diff
@@ -22,6 +23,10 @@ import Data.String (IsString (..))
 import Data.Text qualified as Text
 import GHC.Magic qualified as Magic
 import Prelude qualified as Haskell (String)
+#if MIN_VERSION_base(4,20,0)
+import Prelude (type (~))
+#endif
```

Comment:
God, why do they make us go through this.

Reply:
I have removed all `CPP` that I added other than that one.